src/Engine/Manager/ScriptManager.cpp

```diff
@@ -765,7 +801,7 @@ Component::Script* ScriptManager::CreateScript(const Json::Value& node) {
         std::vector<std::string> names = propertyMapJson.getMemberNames();
         std::string json = propertyMapJson.toStyledString();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Variable 'json' is assigned a value that is never used. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunreadVariable)